.github/workflows/deployment.yml

```
@@ -20,6 +20,9 @@ jobs:
       vite_react_app_mermaid_collect_link: https://dev-app.datamermaid.org
       vite_react_app_mermaid_api_endcpoint: https://dev-api.datamermaid.org/v1/projectsummarysampleevents
       vite_react_app_map_tiler_api_key: L62nVuVTVPFCfHQKVqVk
+      vite_environment: dev
```

actionlint (1.7.7)

input "vite_environment" is not defined in "./.github/workflows/_deploy.yml" reusable workflow. defined inputs are "domain", "environment", "vite_react_app_auth0_audience", "vite_react_app_auth0_client_id", "vite_react_app_auth0_domain", "vite_react_app_map_tiler_api_key", "vite_react_app_mermaid_api_endcpoint", "vite_react_app_mermaid_collect_link"

(workflow-call)